unittests/snapshot_tests.cpp

```diff
@@ -697,13 +701,17 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(test_kv_snapshot, SNAPSHOT_SUITE, snapshot_suites)
             auto snapshot = SNAPSHOT_SUITE::finalize(writer);
 
             auto cfg = chain.get_config();
-            cfg.use_rocksdb_for_disk = rocks_load;
+            if (rocks_load) {
```

Comment:
It looks like rocks_save isn't needed anymore, right?  It also looks like rocks_load will only be true, shouldn't the for loop be:
for (bool rocks_load : { true, false }) {

Reply:
This will be removed in final phase.